code401-challenges/src/main/java/code401/challenges/RepeatedWord/RepeatedWord.java

```diff
@@ -0,0 +1,21 @@
+package code401.challenges.RepeatedWord;
+
+import java.util.HashMap;
+
+public class RepeatedWord {
+
+    public String occurMoreThanOnce(String sentence){
+        String[] wordsArr = sentence.split("\\W+");
+        HashMap<String, Integer> occuranceMap = new HashMap<>();
+        int dummyCnt = 0;
```

Comment:
Instead of creating a counter you could have just inserted any value into the hashmap since the value didn't matter